bot/src/main/kotlin/io/heapy/kotbot/bot/rule/Extensions.kt

```diff
@@ -0,0 +1,13 @@
+package io.heapy.kotbot.bot.rule
+
+/**
+ * Return only single action.
+ */
+@Suppress("NOTHING_TO_INLINE")
+inline fun Action.only(): List<Action> = listOf(this)
```

Comment:
I hope you are not against these extensions. :)

Reply:
I'm not found them necessary, but ok

Reply:
Not something extremely-useful, yup. Still looks bit nicer. :)